draft-ietf-quic-transport.md

```diff
@@ -4084,23 +4088,28 @@ fragmentation on the path.
 A client MUST expand the payload of all UDP datagrams carrying Initial packets
 to at least the smallest allowed maximum packet size (1200 bytes) by adding
 PADDING frames to the Initial packet or by coalescing the Initial packet; see
-{{packet-coalesce}}.  Sending a UDP datagram of this size ensures that the
-network path from the client to the server supports a reasonable Path Maximum
-Transmission Unit (PMTU).  This also helps reduce the amplitude of amplification
-attacks caused by server responses toward an unverified client address; see
+{{packet-coalesce}}.  Similarly, a server MUST expand the payload of all UDP
+datagrams carrying ack-eliciting Initial packets.  Sending UDP datagrams of this
+size ensures that the network path supports a reasonable Path Maximum
+Transmission Unit (PMTU), in both directions.  Additionally, a client padding
+Initial packets helps reduce the amplitude of amplification attacks caused by
+server responses toward an unverified client address; see
 {{address-validation}}.
 
-Datagrams containing Initial packets MAY exceed 1200 bytes if the client
+Datagrams containing Initial packets MAY exceed 1200 bytes if the endpoint
 believes that the network path and peer both support the size that it chooses.
 
 A server MUST discard an Initial packet that is carried in a UDP datagram with a
 payload that is less than the smallest allowed maximum packet size of 1200
 bytes.  A server MAY also immediately close the connection by sending a
 CONNECTION_CLOSE frame with an error code of PROTOCOL_VIOLATION; see
-{{immediate-close-hs}}.
+{{immediate-close-hs}}.  When a client receives an ack-eliciting Initial packet
+that is carried in a UDP datagram with a payload that is less than 1200 bytes,
+that client MAY close the connection by sending a CONNECTION_CLOSE frame.
 
 The server MUST also limit the number of bytes it sends before validating the
-address of the client; see {{address-validation}}.
+address of the client, and MUST pad ack-eliciting Initial packets that it sends;
+see {{address-validation}}.
```

Comment:
```suggestion
address of the client; see {{address-validation}}.
```

I don't think you need this here, since this is described earlier in this very section.